Run3Detector/analysis/utilities/milliqanScheduler.py

```diff
@@ -21,7 +21,10 @@ def __init__(self, inputs, cuts=None, plotter=None):
     def setEvents(self, events):
         self.events = events
         self.cuts.events = self.events
-        self.plotter.events = self.events
+        try:
+            self.plotter.events = self.events
+        except AttributeError as error:
+            print("please be awared that plotter is not being used now")
```

Comment:
Spelling error:  Change "awared" to "aware".

Reply:
Is an exception really the best way to deal with this?

Reply:
what about if self.plotter != None:  self.plotter.events = self.events

> Is an exception really the best way to deal with this?